tests/SDAccel/hls_with_cppkernel_opencl_kernel_int_vector_add_args.cpp

```diff
@@ -0,0 +1,84 @@
+/* Simple OpenCL vector addition using SYCL in OpenCL interoperability
```

Comment:
Most of my comments to `tests/SDAccel/hls_with_cppkernel_opencl_kernel_ap_vector_add_args.cpp`